src/error/GraphQLError.ts

```diff
@@ -78,13 +113,17 @@ export class GraphQLError extends Error {
 
   constructor(
     message: string,
-    nodes?: ReadonlyArray<ASTNode> | ASTNode | null,
-    source?: Maybe<Source>,
-    positions?: Maybe<ReadonlyArray<number>>,
-    path?: Maybe<ReadonlyArray<string | number>>,
-    originalError?: Maybe<Error & { readonly extensions?: unknown }>,
-    extensions?: Maybe<GraphQLErrorExtensions>,
-  ) {
+    nodes?: GraphQLErrorArgs['nodes'],
+    source?: GraphQLErrorArgs['source'],
+    positions?: GraphQLErrorArgs['positions'],
+    path?: GraphQLErrorArgs['path'],
+    originalError?: GraphQLErrorArgs['originalError'],
+    extensions?: GraphQLErrorArgs['extensions'],
+  );
```

Comment:
Please keep the old type "as is".
It's temporary anyway until we remove the position-based one in the next release.
BTW, you can add deprecation notice for this signature.

Reply:
done